internal/core/items/text.rs

```diff
@@ -557,7 +558,8 @@ impl TextInput {
         let mut grapheme_cursor =
             unicode_segmentation::GraphemeCursor::new(last_cursor_pos, text.len(), true);
 
-        let font_height = window.text_size(self.unresolved_font_request(), "", None).height;
+        let font_height = window.text_size(self.unresolved_font_request(), " ", None).height
+            / window.scale_factor();
```

Comment:
I don't think the division by the  scale_factor is needed, `text_size` should IMO return logical coordinates.

And I guess the space is not needed and you can provide an empty string with your fix to `text_size`?

Reply:
yeah. i already found that too. without that mistake the gl backend works on high res too.